From 7bd280f3cc07208760759c98bbfbac3f9d28f77c Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 5 Mar 2021 08:58:29 -0800
Subject: [PATCH] pollGtk: Drop volatile qualifier

glib-2.0 has changed the function interfaces as well and do not expect
it to be volatile. GCC 11 complains about it

glib/gatomic.h:112:5: error: argument 2 of '__atomic_load' discards 'volatile' qualifier [-Werror=incompatible-pointer-types]

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 open-vm-tools/lib/pollGtk/pollGtk.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/open-vm-tools/lib/pollGtk/pollGtk.c b/open-vm-tools/lib/pollGtk/pollGtk.c
index 6d95bf01..a941e7a7 100644
--- a/open-vm-tools/lib/pollGtk/pollGtk.c
+++ b/open-vm-tools/lib/pollGtk/pollGtk.c
@@ -1473,7 +1473,7 @@ PollGtkBasicCallback(gpointer data) // IN: The eventEntry
 void
 Poll_InitGtk(void)
 {
-   static volatile gsize inited = 0;
+   static gsize inited = 0;
 
    static const PollImpl gtkImpl =
    {
-- 
2.30.1

